src/style.scss

```diff
@@ -14,16 +20,17 @@ body {
     margin-bottom: 30px;
     margin-left: 40px;
     h1 {
-      font-size: modular-scale(2);
       small {
```

Comment:
I could not see there the module-scale was working at all. Perhaps it was installed at some point and then removed.